WalletWasabi.Gui/Controls/WalletExplorer/TransactionBroadcasterViewModel.cs

```diff
@@ -144,7 +144,7 @@ public override void OnOpen()
 		{
 			if (Disposables != null)
 			{
-				throw new Exception("TransactionBroadcaster was opened before it was closed.");
+				throw new Exception("Transaction Broadcaster was opened before it was closed.");
```

Comment:
Is it wise to change the exception message like this?

Reply:
In the `TransactionViewerViewModel.cs` file `TransactionViewer` was written `Transaction Viewer was opened...` so I followed that, I don't know if we should change it or not but we have to be consistent.